### PR TITLE
Release google_symptoms

### PR DESCRIPTION
**Pull request by korlaxxalrok**
### Description
Prepping google_symptoms for release.

### Changelog
Itemize code/test/documentation changes and files added/removed.
- Add production params.

### CODE REVIEW

File: ansible/templates/google_symptoms-params-prod.json.j2 [hunk 1]
```diff
@@ -0,0 +1,7 @@
+{
+  "export_start_date": "2020-02-20",
+  "static_file_dir": "./static",
+  "export_dir": "./common/covidcast/receiving/safegraph_patterns",
```
Comment (krivard) [inline ansible/templates/google_symptoms-params-prod.json.j2:4]:
```suggestion
  "export_dir": "./common/covidcast/receiving/google_symptoms",
```

Comment (korlaxxalrok) [reply ansible/templates/google_symptoms-params-prod.json.j2]: Oh! Should be `"/common/covidcast/receiving/google_symptoms"` (sans `.`)?

(Tried to un-resolve this so I could suggest, but I think I'm doing it wrong.

Comment (korlaxxalrok) [reply ansible/templates/google_symptoms-params-prod.json.j2]: @krivard I wasn't sure how to properly update this change. See if it makes sense to you.

File: ansible/templates/google_symptoms-params-prod.json.j2 [hunk 2]
```diff
@@ -0,0 +1,7 @@
+{
+  "export_start_date": "2020-02-20",
+  "static_file_dir": "./static",
+  "export_dir": "./common/covidcast/receiving/google_symptoms",
```
Comment (krivard) [inline ansible/templates/google_symptoms-params-prod.json.j2:4]:
```suggestion
  "export_dir": "/common/covidcast/receiving/google_symptoms",
```

**Pull request merged**